### ISSUE TITLE
[Feature Request] support GFM checkboxes to detect the scope

### ISSUE THREAD

**Issue by zakhenry** [ciena-blueplanet/pr-bumper#46]
It would be handy to be able to have a list of options defined in a pull request template that the requester can just tick rather than having to remember the syntax.

For example in the PR template

### Check the scope of this pr:
- [ ] #patch# - bugfix, dependency update
- [x] #minor# - new feature, backwards compatible
- [ ] #major# - major feature, probably breaking API
- [ ] #breaking# - any change that breaks the API

I think this would be best placed in the catch case where more than one scope is detected - https://github.com/ciena-blueplanet/pr-bumper/blob/master/lib/utils.js#L160

Here is an example regex I put together to test the concept
```js
const [match, scope] = /-\s\[x\].*?#(\w+)#/g.exec(`
### Check the scope of this pr:
- [ ] #patch# - bugfix, dependency update
- [x] #minor# - new feature, backwards compatible
- [ ] #major# - major feature, probably breaking API
- [ ] #breaking# - any change that breaks the API`);
> match
"- [x] #minor#"
> scope
"minor"
```

Happy to create a PR for it, just want to know if there is interest in the feature before I bother

**Comment by sglanzer** [comment]
This is a cool idea, definitely would help discoverability of the feature on a lot of the repos we use pr-bumper on (as opposed to a failed build, then informing someone that they need to add tags)

**Comment by zakhenry** [comment]
@sglanzer implemented in #47
